sites/cheerpx/src/content/docs/12-reference/CheerpX.Linux/04-setCustomConsole.md

```diff
@@ -7,23 +7,23 @@ description: Configure custom console behavior for handling keyboard input and o
 namespace CheerpX {
 	class Linux {
 		setCustomConsole(
-			writeFunc: (buf: Buffer) => void,
+			writeFunc: (buf: Uint8Array, vt: number) => void,
 			cols: number,
 			rows: number
-		): (keyCode: number) => void;
+		): client.EventListener;
```

Comment:
This is wrong, EventListener is just a placeholder for an unknown callback. We _know_ the type and should tell the user. The previous version was correct.